app/components/SearchTableLayout.tsx

```diff
@@ -22,6 +22,17 @@ export const SearchTableLayoutComponent: React.FunctionComponent<Props> = (
     isLoading
   } = props;
 
+  const noSearchResults =
+    body.props.children.length === 0 ||
+    body?.props?.children[0]?.length === 0 ? (
+      <span id="no-search-results">
+        <Alert variant="secondary">
+          <p>&quot;No matching results to show.&quot;</p>
+        </Alert>
+        <br />
+      </span>
```

Comment:
- The quotes are unnecessary. (sorry - I can see this was probably a result of a rather literal interpretation of what I wrote in the card 😊)

- To remove some of the awkward extra whitespace shown below, this whole span block can be simplified to:
```
    <Alert variant="secondary">
      No matching results to show.
    </Alert>
```
Only the [Alert and simple text content is needed](https://react-bootstrap.netlify.app/components/alerts/) - the span (which is an [inline element](https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements#Inline_vs._block-level_elements_a_demonstration)) is unnecessary and can be removed:

before:
<img width="596" alt="Screen Shot 2020-08-25 at 9 06 28 AM" src="https://user-images.githubusercontent.com/5522075/91199114-471ef600-e6b2-11ea-8fb9-3a49d66e25f3.png">

after:
<img width="587" alt="Screen Shot 2020-08-25 at 9 07 32 AM" src="https://user-images.githubusercontent.com/5522075/91199204-6ae23c00-e6b2-11ea-87f8-09a55104b244.png">